care/utils/serializer/phonenumber_ispossible_field.py

```diff
@@ -0,0 +1,18 @@
+import phonenumbers
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'phonenumbers'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5343021)